src/common.rs

```diff
@@ -1029,6 +1029,41 @@ pub fn print_transaction_status(
     );
 }
 
+#[cfg(target_os = "macos")]
+pub async fn save_access_key_to_macos_keychain(
+    network_config: crate::config::NetworkConfig,
+    key_pair_properties: crate::common::KeyPairProperties,
+    account_id: &str,
+) -> crate::CliResult {
+    let buf = format!(
+        "{}",
+        serde_json::json!({
+            "master_seed_phrase": key_pair_properties.master_seed_phrase,
+            "seed_phrase_hd_path": key_pair_properties.seed_phrase_hd_path.to_string(),
+            "account_id": account_id,
+            "public_key": key_pair_properties.public_key_str,
+            "private_key": key_pair_properties.secret_keypair_str,
+        })
+    );
```

Comment:
I believe `cargo clippy` should catch such cases. Please, use it.

```suggestion
    let buf = serde_json::json!({
            "master_seed_phrase": key_pair_properties.master_seed_phrase,
            "seed_phrase_hd_path": key_pair_properties.seed_phrase_hd_path.to_string(),
            "account_id": account_id,
            "public_key": key_pair_properties.public_key_str,
            "private_key": key_pair_properties.secret_keypair_str,
        }).to_string();
```